loader/pom.xml

```diff
@@ -3,7 +3,7 @@
 
   <groupId>org.lucee</groupId>
   <artifactId>lucee</artifactId>
-  <version>5.2.9.31-SNAPSHOT</version>
+  <version>5.2.9.32-TEN24</version>
```

Comment:
@migueltarga should we name it 5.2.9.31-TEN24002 because it's still derived from lucee 5.2.9.31?

Reply:
I realized after when I created the branch, the branch name is correct... 😅